packages/reown_appkit/example/base/pubspec.yaml

```diff
@@ -12,14 +12,14 @@ environment:
 dependencies:
   bs58: ^1.0.2
   cupertino_icons: ^1.0.8
-  eth_sig_util: ^0.0.9
+  eth_sig_util_plus: ^0.0.10
```

Comment:
This is a hard fork of https://pub.dev/packages/eth_sig_util. I am now maintaining the fork. The idea of its publisher (Dart Anchor)[https://github.com/dart-anchor] (which is also founded by me) is *Builds solid foundation packages for the Dart ecosystem*.